app/__init__.py

```diff
@@ -32,67 +21,33 @@
 dropzone = Dropzone()
 heroku = Heroku()
 
-
-# Set up a Flask app
 def create_app(config_class=Config):
     app = Flask(__name__)
-    # with information specified in the config.py
+    app.config['SECRET_KEY'] = 'your-secret-key'
     app.config.from_object(config_class)
-    # connect to the database
+
+
     db.init_app(app)
     migrate.init_app(app, db)
-    db.create_all(app=app)
-    # Initialise additional functionality
     login.init_app(app)
     mail.init_app(app)
     bootstrap.init_app(app)
     moment.init_app(app)
     dropzone.init_app(app)
     heroku.init_app(app)
 
-    # Add project-specific functionality
-    # Main page - Dashboard
-    from app.main import bp as main_bp
-    app.register_blueprint(main_bp)
-    # Upload directory to hold files
-    os.makedirs(app.config['UPLOADED_PATH'], exist_ok=True)
+    # Set SQLALCHEMY_DATABASE_URI to PostgreSQL URL string
+    app.config['SQLALCHEMY_DATABASE_URI'] = f"postgresql://{os.environ['POSTGRES_USER']}@localhost:5432/{os.environ['POSTGRES_DB']}"
```

Comment:
@Lucrezia-Cester I've just tried to run the app and I receive the following error:

![image](https://user-images.githubusercontent.com/67117138/233612395-e7817b9f-3dca-4d88-9dc2-65702b9f907b.png)

What is the reasoning behind this change? I'm wondering if I've missed a setup step that's causing it to fail...